crypt/cipher.go

```diff
@@ -317,8 +316,10 @@ func (c *cipher) obfuscateSegment(plaintext string) string {
 		return ""
 	}
 	var dir int
-	// Calculate a simple rotation based on the filename
+	// Calculate a simple rotation based on the filename and
+	// the dataKey
 	for i:= 0; i<len(plaintext); i++ { dir += int(plaintext[i]) }
+	for i:= 0; i<len(c.dataKey); i++ { dir += int(c.dataKey[i]) }
```

Comment:
you want to use c.nameKey for this - dataKey is for the actual data.  Note that using it in a simple cipher like this risks leaking the key and you definitely don't want to leak the dataKey.